schedule/kernel/agama_gfx_drivers.yaml

```diff
@@ -3,9 +3,9 @@ name: agama-gfx-usb-drivers
 vars:
     AGAMA: 1
     AGAMA_PRODUCT_ID: SLES
```

Comment:
Also remove `AGAMA_PRODUCT_ID: SLES` from all schedules. This is defined in product medium settings.